QtCollider/primitives/prim_misc.cpp

```diff
@@ -168,7 +168,7 @@ QC_LANG_PRIMITIVE( Qt_SetGlobalPalette, 1, PyrSlot *r, PyrSlot *a, VMGlobals *g
 {
   if( !QcApplication::compareThread() ) return QtCollider::wrongThreadError();
 
-  QPalette p = QtCollider::get( a );
+  QPalette p = (QPalette&&) QtCollider::get(a);
```

Comment:
well, un-safe type-casts are never safe (by definition).

fwiw, this moves (=invalidates) the original object, while the old code copied it.